chat-sdk/src/main/java/com/amazon/connect/chat/sdk/repository/ChatService.kt

```diff
@@ -728,19 +728,32 @@ class ChatServiceImpl @Inject constructor(
         fetchTranscriptWith(startPosition)
     }
 
+    private fun isItemInInternalTranscript(Id: String?): Boolean {
+        if (Id == null) return false
+        for (item in internalTranscript.reversed()) {
+            if (item.id == Id) {
+                return true
+            }
+        }
+        return false
+    }
+
     private suspend fun fetchTranscriptWith(startPosition: StartPosition?) {
         getTranscript(startPosition = startPosition,
             scanDirection = ScanDirection.FORWARD,
             sortKey = SortKey.ASCENDING,
-            maxResults = 30,
+            maxResults = 100,
```

Comment:
what made us to increase the maxResults ? will not increase the response object size & latency ?